NumLib/DOF/DOFTableUtil.cpp

```diff
@@ -30,4 +44,60 @@ std::vector<GlobalIndexType> getIndices(
     return indices;
 }
 
+double norm(GlobalVector const& x, unsigned const global_component,
```

Comment:
This implementation of a norm function is very special. It mixes the usual norm computation with fetching the values. Is there a special reason to couple these two different tasks?